mcp/motors/cryovalves.c

```diff
@@ -33,6 +33,11 @@
 #include "tx.h"
 #include "actuators.h"
 #include "cryovalves.h"
+#include "blast.h"
+
+#ifndef MAX
+#define MAX(a, b) ((a) > (b) ? (a) : (b))
+#endif
 
```

Comment:
It's out of the scope of this ticket, but we may want to have a dedicated place for such generic macros. Now it is being defined in multiple places:
```
$ grep -inr '#define MAX(.*)' .
./common/include/FIFO.h:39:#define MAX(a, b) ((a) > (b) ? (a) : (b))
./common/include/bitserver.h:55:#define MAX(a, b) ((a) > (b) ? (a) : (b))
./common/include/blast.h:173:#define max(a, b) ((a) >= (b) ? (a) : (b))
./common/logger.c:41:#define MAX(a, b) ((a) > (b) ? (a) : (b))
./external_libs/socat/mytypes.h:20:#define Max(x,y) ((x)>=(y)?(x):(y))
./groundhog/blast/groundhog.h:19:#define MAX(a, b) ((a) > (b) ? (a):(b))
./liblinklist/linklist_connect.h:71:#define MAX(a,b) ((a) > (b) ? (a) : (b))
./mcp/motors/cryovalves.c:39:#define MAX(a, b) ((a) > (b) ? (a) : (b))
```

Reply:
Issue #26 created.